tests/python/unittest/test_auto_scheduler_layout_rewrite.py

```diff
@@ -166,5 +166,6 @@ def test_correctness_layout_rewrite_insert_transform_stage():
 
 if __name__ == "__main__":
     test_apply_steps_with_layout_rewrite()
-    test_correctness_layout_rewrite_rewrite_for_preTransformed()
-    test_correctness_layout_rewrite_insert_transform_stage()
+    # Disable for now due to being flaky on i386
+    # test_correctness_layout_rewrite_rewrite_for_preTransformed()
+    # test_correctness_layout_rewrite_insert_transform_stage()
```

Comment:
In case this is unstable only on `i386`, is there a possibility to use a `skipif` clause rather than disabling it everywhere?

Reply:
It's only a temporary measure, i386 tests should be fixed soon.

Reply:
Yes, I'm working on this. :)

Reply:
Understood.

Reply:
That is #6830 I guess